comp129-spr19/project2-osvaldo-s-children
# Experimenting schema

This branch's primary purpose was to create a database within Django for the donation/events_needs page to store data. 

To test out the capabilities of this branch:
 1. Go into the root of our website in the terminal, donatefoods, and run the server. 
 2. Type python manage.py shell
 2.  call event1.food_bank_event, to see the stored item or to create an item to be stored, carry on as followed.
 2.  Type from events_needs.models import FoodBanks, FoodBankEvents 
 3. Then type, for example, event1 = FoodBankEvents(food_bank_event = "Strawberry Fest", food_bank_date = "2018-07-17", food_bank_description = "Strawberries canned, fresh, or frozen, please...with a side of chocolate") 

Note: order matters as the information is relative to how the columns are organized in the module. Therefore, for the respective model/table be sure the format of inserted information is correctly inserted

 4.  event1.save() 
 5. call event1.food_bank_event, to see stored item

To exit the python interactive terminal type quit() or quit.
